core/src/main/java/io/questdb/cutlass/text/types/IPv4Adapter.java

```diff
@@ -45,16 +45,24 @@ public int getType() {
 
     @Override
     public boolean probe(DirectUtf8Sequence text) {
-        if (text.size() < 7)
+        if (text.size() < 7) {
             return false;
+        }
+        boolean noDots = true;
         if (Numbers.notDigit(text.byteAt(0))) {
             if (text.byteAt(0) != '.') {
                 return false;
+            } else {
+                noDots = false;
             }
         }
 
         try {
-            Numbers.parseIPv4(text);
+            if (noDots) {
+                Numbers.parseIPv4(text);
+            } else {
+                Numbers.parseInt(text);
```

Comment:
this is strange logic, if there dots in the text - we should be treating text as ipv4, not int?

Reply:
is there test?

Reply:
> this is strange logic, if there dots in the text - we should be treating text as ipv4, not int?

That branch was taken since there was no test with first non-digit byte. Fixed the logic and improved `IODispatcherTest.testImportIPv4` in da47fd116c6fc66ce813af56d91d01000480a00e